beamer/tests/contracts/test_request_manager_fees.py

```diff
@@ -246,16 +246,73 @@ def test_different_fees(request_manager, token, claim_period, claim_stake):
     assert token.balanceOf(claimer) == amount * 2 + lp_fee_1 + lp_fee_2
 
 
-def test_ppm_out_of_bound(request_manager, token):
+def test_ppm_out_of_bound(request_manager):
     ppm = 999_999
 
-    request_manager.updateToken(token.address, 0, 0, ppm, 0)
-    request_manager.updateToken(token.address, 0, 0, 0, ppm)
+    request_manager.updateFees(0, ppm, ppm)
 
     ppm = 1_000_000
 
     with ape.reverts("Maximum PPM of 999999 exceeded"):
-        request_manager.updateToken(token.address, 0, 0, ppm, 0)
+        request_manager.updateFees(0, ppm, 0)
 
     with ape.reverts("Maximum PPM of 999999 exceeded"):
-        request_manager.updateToken(token.address, 0, 0, 0, ppm)
+        request_manager.updateFees(0, 0, ppm)
+
+
+@pytest.mark.parametrize("transfer_cost", [int(400e12)])
+def test_min_lp_fee(request_manager, token, chain_params, min_fee_ppm, token_params):
+    target_chain_params = chain_params[0], int(3e12), 750_000
+    request_manager.updateChain(999, *target_chain_params)
+
+    source_chain_cost = (1_000_000 - chain_params[2]) * chain_params[1]
+    target_chain_cost = target_chain_params[2] * target_chain_params[1]
+    min_lp_fee = (
+        (source_chain_cost + target_chain_cost)
+        * token_params[1]
+        * (1_000_000 + min_fee_ppm)
+        / 1e30
+    )
+
+    assert min_lp_fee == request_manager.minLpFee(999, token.address)
+
+
+@pytest.mark.parametrize("transfer_cost", [int(400e12)])
+def test_total_fee(request_manager, token, chain_params):
+    target_chain_params = chain_params[0], int(3e12), 750_000
+    request_manager.updateChain(999, *target_chain_params)
+
+    min_lp_fee = request_manager.minLpFee(999, token.address)
+    # find the value where lpFee == minLpFee to test with values lower and higher
+    conjunction_amount = int(min_lp_fee * 1_000_000 / (request_manager.lpFeePPM()))
+    test_amounts = [int(conjunction_amount / 2), conjunction_amount * 2]
```

Comment:
Nitpick: instead of doing `int(...)` conversion, you can use the floor division operator `//`: `conjunction_amount // 2`.